services/121-service/src/payments/transactions/transaction.repository.ts

```diff
@@ -66,7 +66,7 @@ export class TransactionScopedRepository extends ScopedRepository<TransactionEnt
         'transaction.updated AS "updated"',
         'transaction.payment AS payment',
         'r."registrationProgramId"',
-        'r."referenceId" as "referenceId"',
+        'r."referenceId" as "registrationReferenceId"',
```

Comment:
Maybe this 'aliasing' is not really enough;
But maybe the actual migration can be made separately...?

Reply:
The database/table-column-name for example...

Reply:
Not needed :) We talked about this already.